schema-engine/connectors/sql-schema-connector/src/flavour/postgres/native/mod.rs

```
@@ -269,8 +269,8 @@ pub(super) fn get_circumstances(state: &State) -> Option<BitFlags<Circumstances>
     }
 }
 
-pub(super) fn get_default_schema(state: &State) -> Option<&str> {
-    state.params().map(|params| params.url.schema())
+pub(super) fn get_default_schema(state: &State) -> &str {
+    state.get_unwrapped_params().url.schema()
```

How is this safe?

---

this works because we only call this once we already have connected to the database, in the existing code we do `unwrap_or("public")`, but I think it's better to have a panic rather than a potentially unexpected fallback when the function is called before the connector is properly initialized

---

It'd be nice to eventually get rid of the 'uninitialized' state so that we don't have to do stuff like this, but we still need it for now.